src/matlab/homotopy_solver.m

```diff
@@ -63,15 +63,8 @@
 end
 p_val = [model.p_val(:);x0(:); lambda00(:)];
 
-% TODO remove try!
-try
-    complementarity_stats = [full(comp_res(w0, p_val))];
-catch
-    w0 = w0(1:length(model.w));
-    w0
-    p_val
-    complementarity_stats = [full(comp_res(w0, p_val))];
-end
+complementarity_stats = [full(comp_res(w0, p_val))];
```

Comment:
comp_res still computes $\ell_1$ without the cross terms, right? 
nice that the stupid try-catch is resolved.

Reply:
Yep, it still does, I haven't quite finished that, but hopefully will today in a follow on PR. As for the formatting I made it look nice in my command window so it seems somehow our machines are handling tabs differently. I'm going to look if matlab has a nice table output formatter, and if it does switch to using that.

Reply:
Lets get the current PR through as the inclusion part is anyway good and let the verbose be part of the next one.